doc/releases/changelog-0.36.0.md

```diff
@@ -303,68 +383,90 @@
 
   Note that the grouping indices refer to the lists returned by `Sum.terms()`, not `Sum.operands`.
 
-* Added new function `qml.operation.convert_to_legacy_H` to convert `Sum`, `SProd`, and `Prod` to `Hamiltonian` instances.
+* A new function called `qml.operation.convert_to_legacy_H` that converts `Sum`, `SProd`, and `Prod` to `Hamiltonian` instances has been added.
+  This function is intended for developers and will be removed in a future release without a
+  deprecation cycle.
   [(#5309)](https://github.com/PennyLaneAI/pennylane/pull/5309)
 
 * The `qml.is_commuting` function now accepts `Sum`, `SProd`, and `Prod` instances.
   [(#5351)](https://github.com/PennyLaneAI/pennylane/pull/5351)
 
-* Operators can now be left multiplied `x * op` by numpy arrays.
+* Operators can now be left-multiplied by NumPy arrays (i.e., `arr * op`).
   [(#5361)](https://github.com/PennyLaneAI/pennylane/pull/5361)
 
-* A new class `qml.ops.LinearCombination` is introduced. In essence, this class is an updated equivalent of `qml.ops.Hamiltonian`
-  but for usage with new operator arithmetic.
-  [(#5216)](https://github.com/PennyLaneAI/pennylane/pull/5216)
-
-* The generators in the source code return operators consistent with the global setting for
-  `qml.operator.active_new_opmath()` wherever possible. `Sum`, `SProd` and `Prod` instances
-  will be returned even after disabling the new operator arithmetic in cases where they offer
-  additional functionality not available using legacy operators.
+* `op.generator()`, where `op` is an `Operator` instance, now returns operators 
+  consistent with the global setting for `qml.operator.active_new_opmath()` wherever possible. 
+  `Sum`, `SProd` and `Prod` instances will be returned even after disabling the 
+  new operator arithmetic in cases where they offer additional functionality not 
+  available using legacy operators.
   [(#5253)](https://github.com/PennyLaneAI/pennylane/pull/5253)
   [(#5410)](https://github.com/PennyLaneAI/pennylane/pull/5410)
   [(#5411)](https://github.com/PennyLaneAI/pennylane/pull/5411)
   [(#5421)](https://github.com/PennyLaneAI/pennylane/pull/5421)
 
-* A new `Prod.obs` property is introduced to smoothen the transition of the new operator arithmetic system.
-  In particular, this aims at preventing breaking code that uses `Tensor.obs`. This is immediately deprecated.
-  Moving forward, we recommend using `op.operands`.
+* `Prod` instances temporarily had a new `obs` property, which helped smoothen the 
```

Comment:
@trbromley this was an interesting entry in the changelog lol. How does it read?